keps/sig-scheduling/1923-prefer-nominated-node/README.md

```diff
@@ -198,6 +199,11 @@ in back-to-back releases.
 - Gather feedback from developers and surveys.
 - The feature is guarded by a feature flag, and will be enabled by default in beta.
 
+#### GA
+
+- No negtive feedback.
+- The feature gate `PreferNominatedNode` will graduate to GA.
```

Comment:
is there a link to the testgrid for integration and benchmark tests that you can include here?

Reply:
I searched in the testgrid (https://k8s-testgrid.appspot.com/sig-scheduling) and haven't found the link of the integration test. 

Also the perf dash seems cannot tell much details about the individual metrics (https://perf-dash.k8s.io/#/?jobname=scheduler-perf-benchmark&metriccategoryname=Scheduler&metricname=BenchmarkResults&benchmark=BenchmarkPerfScheduling%2FSchedulingBasic%2F500Nodes-8&metricName=time)

Reply:
https://k8s-testgrid.appspot.com/presubmits-kubernetes-blocking#pull-kubernetes-integration

Reply:
thanks for showing me the link, will update.